CMakeLists.txt

```diff
@@ -333,6 +333,15 @@ if(HPX_WITH_AUTOMATIC_SERIALIZATION_REGISTRATION)
   hpx_add_config_define(HPX_HAVE_AUTOMATIC_SERIALIZATION_REGISTRATION)
 endif()
 
+hpx_option(HPX_WITH_SIGNAL_HANDLERS BOOL
+  "Catches unhandled exceptions and produces debugging output (default: ON)"
+  ON
+  ADVANCED)
+
+if(NOT HPX_WITH_SIGNAL_HANDLERS)
+  add_definitions(-DHPX_DISABLE_SIGNAL_HANDLERS)
+endif()
```

Comment:
The convention in HPX is to name the PP constant corresponding to a `HPX_WITH_FOO` configuration setting as `HPX_HAVE_FOO`. Could you change this name accordingly, please? I'd suggest to name it `HPX_HAVE_NO_SIGNAL_HANDLERS` or somesuch. Please also align the sames such that the configuration option is named similarily (i.e. `HPX_WITH_NO_SIGNAL_HANDLERS`).